StuDash/settings.py

```diff
@@ -29,6 +29,8 @@
     # Application apps
     "main",
     "accounts",
+    "forumMessages",
+    "forumMessages.apps.ForumMessagesConfig",
```

Comment:
Why do you need that line?

Reply:
I think you don't need this line, but I will merge it anyhow

Reply:
It was indeed unnecessary, even caused a duplication error in some scenarios.
I've fixed this on #49 .